core/backend/src/test/element/ExcludedElements.test.ts

```diff
@@ -57,6 +57,7 @@ describe("ExcludedElements", () => {
       // Unless compressed Ids explicitly requested, the Ids are always decompressed regardless of how they are stored.
       // This is to preserve compatibility with older front-ends that don't understand the compressed Ids; it's an unfortunate default.
       // ###TODO change the default in iModel.js 3.0.
+      // CMS: PMC??
```

Comment:
@pmconne this is unrelated to this specific PR but that comment above makes it seem like we should have made a change here for 3.0. Has that change been made or should it before we release?

Reply:
Options:
1. Keep APIs the same and simply change `compressExcludedElementIds` to default to true instead of true. This will silently break any code relying on the previous default. That seems undesirable.
2. Change the API to replace `compressedExcludedElementIds` with `decompressExcludedElementIds`. This will produce a compiler error for existing code forcing them to decide whether they want the old or new behavior. It is a breaking API change (too late for that, no?) and would require native changes and a new addon.
3. Keep the existing default behavior and change nothing. The primary way this function gets called is via `IModelConnection.views.load` which overrides the default to always compress the Ids.

I vote for #3.

Reply:
Okay, 3 works with me as well. I'll remove the comment then.

Reply:
Removed and resolving the comment.